pkg/storage/cacher.go

```diff
@@ -772,7 +772,7 @@ func (c *cacheWatcher) process(initEvents []watchCacheEvent, resourceVersion uin
 		}
 		// only send events newer than resourceVersion
 		if event.ResourceVersion > resourceVersion {
-			c.sendWatchCacheEvent(event)
+			c.sendWatchCacheEvent(&event)
```

Comment:
ok, just realized sendWatchCacheEvent copies the object and constructs a watch.Event per watcher, so that's fine